scripts/installer_test.go

```diff
@@ -0,0 +1,339 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+// +build windows
+// +build ignore
+
+package main
+
+import (
+	"archive/zip"
+	"bufio"
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/zeebo/errs"
+	"golang.org/x/sync/errgroup"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
+
+	"storj.io/storj/private/testcontext"
+)
+
+var (
+	releaseMSIPath string
+	// TODO: make this more dynamic and/or use versioncontrol server?
+	// (NB: can't use versioncontrol server until updater process is added to response)
+	oldRelease  = "v0.26.2"
+	msiBaseArgs = []string{
+		"/quiet", "/qn",
+		"/norestart",
+	}
+
+	msiPathFlag = flag.String("msi", "", "path to the msi to use")
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	if *msiPathFlag == "" {
+		log.Fatal("no msi passed, use `go test ... -args -msi \"<msi path (using \\ seperator)>\"`")
+	}
+
+	tmp, err := ioutil.TempDir("", "release")
```

Comment:
Why don't we use `testcontext` for creating this temp dir and let it take care for the cleanup?

Reply:
A testcontext can only be created inside of a test. This is the `TestMain` function and it doesn't receive a `*testing.T` to pass to `textcontext.New`

Reply:
Good point. You are right.